soh/soh/SohMenuBar.cpp

```diff
@@ -0,0 +1,1448 @@
+            UIWidgets::Tooltip("Activates multi-sample anti-aliasing when above 1x up to 8x for 8 samples for every pixel");
+            // OTRTODO: fix this
+            // LUS::SetMSAALevel(CVarGetInteger("gMSAAValue", 1));
+        #endif
+
+            { // FPS Slider
+                const int minFps = 20;
+                static int maxFps;
+                if (LUS::Context::GetInstance()->GetWindow()->GetWindowBackend() == LUS::WindowBackend::DX11) {
+                    maxFps = 360;
+                } else {
+                    maxFps = LUS::Context::GetInstance()->GetWindow()->GetCurrentRefreshRate();
+                }
+                int currentFps = fmax(fmin(OTRGlobals::Instance->GetInterpolationFPS(), maxFps), minFps);
+            #ifdef __WIIU__
+                UIWidgets::Spacer(0);
+                // only support divisors of 60 on the Wii U
+                if (currentFps > 60) {
+                    currentFps = 60;
+                } else {
+                    currentFps = 60 / (60 / currentFps);
+                }
+
+                int fpsSlider = 1;
+                if (currentFps == 20) {
+                    ImGui::Text("FPS: Original (20)");
+                } else {
+                    ImGui::Text("FPS: %d", currentFps);
+                    if (currentFps == 30) {
+                        fpsSlider = 2;
+                    } else { // currentFps == 60
+                        fpsSlider = 3;
+                    }
+                }
+                if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                    UIWidgets::DisableComponent(ImGui::GetStyle().Alpha * 0.5f);
+                }
+
+                if (ImGui::Button(" - ##WiiUFPS")) {
+                    fpsSlider--;
+                }
+                ImGui::SameLine();
+                ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+
+                UIWidgets::Spacer(0);
+
+                ImGui::SliderInt("##WiiUFPSSlider", &fpsSlider, 1, 3, "", ImGuiSliderFlags_AlwaysClamp);
+
+                ImGui::SameLine();
+                ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+                if (ImGui::Button(" + ##WiiUFPS")) {
+                    fpsSlider++;
+                }
+
+                if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                    UIWidgets::ReEnableComponent("");
+                }
+                if (fpsSlider > 3) {
+                    fpsSlider = 3;
+                } else if (fpsSlider < 1) {
+                    fpsSlider = 1;
+                }
+
+                if (fpsSlider == 1) {
+                    currentFps = 20;
+                } else if (fpsSlider == 2) {
+                    currentFps = 30;
+                } else if (fpsSlider == 3) {
+                    currentFps = 60;
+                }
+                CVarSetInteger("gInterpolationFPS", currentFps);
+                LUS::Context::GetInstance()->GetWindow()->GetGui()->RequestCvarSaveOnNextTick();
```

Comment:
```suggestion
                LUS::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesOnNextTick();
```

Reply:
Wonder how it's building without this change.

Reply:
ifdef'd behind wii u

Reply:
fixed https://github.com/HarbourMasters/Shipwright/pull/2881/commits/29016d186ff3efeacf675e43eac5df39a8224175